docs/control-structures.rst

```diff
@@ -196,6 +196,33 @@ can still return a value to the caller by use of the ``return`` statement.
         }
     }
 
+.. index:: ! literal suffix; suffix call syntax
+.. _function-calls-suffix-calls:
+
+Suffix Calls
+------------
+
+For free functions designated as :ref:`literal suffix<literal_suffixes>` definitions there is
+another way to perform a function call when the argument is a literal.
+The function can be used as a suffix: ``42 suffix``.
+
+For single-parameter suffixes, this call syntax is equivalent to passing the literal as an argument
+to an internal function call.
+For two-parameter suffixes the :ref:`fractional decomposition<fractional_decomposition>` is performed
+first to obtain the two input values.
+
+See the section on :ref:`calling suffix functions<calling_suffix_functions>` for more information
+on the limitations of this call syntax.
+
+.. note::
+    Suffix calls should not be confused with the application of :ref:`denomination suffixes<denominations>`,
+    which use the same syntax, but simply modify the literal without an actual function call.
+
+    There is only a limited number of denominations and their names cannot be reused by literal
+    suffixes.
+    The only exceptions are ``finney`` and ``szabo``, which used to be denominations but are now
```

Comment:
just curious now: what about the other denominations? e.g. ``wei`` and ``gwei``? we don't have those in solidity?

Reply:
We still have `wei` and `gwei`. It's only `finney` and `szabo` that were removed, and (what's relevant here) they're not reserved so you can use them for your suffixes. This is not the case with `year` for example, which was also removed but is reserved.